stdlib/universe/range_nsecs_test.flux

```diff
@@ -0,0 +1,353 @@
+package universe_test
+
+
+import "array"
+import "testing"
+
+
+option now = () => 2030-01-01T00:00:00Z
+
+testcase range_nsecs_bare_last {
+input = array.from(
+    rows: [
+        {_time: 2021-01-01T00:00:01.000000001Z, _field: "foo", _value: 1.0,_measurement: "mm"}, 
+        {_time: 2021-01-01T00:00:01.000000002Z, _field: "foo", _value: 2.0,_measurement: "mm"}, 
+        {_time: 2021-01-01T00:00:01.000000005Z, _field: "foo", _value: 3.0,_measurement: "mm"}, 
+        {_time: 2021-01-01T00:00:01.000000024Z, _field: "foo", _value: 11.24,_measurement: "mm"}, 
+        {_time: 2021-01-01T00:00:01.000000009Z, _field: "bar", _value: 4.0,_measurement: "mm"}, 
+        {_time: 2021-01-01T00:00:01.000000011Z, _field: "bar", _value: 5.0,_measurement: "mm"}, 
+        {_time: 2021-01-01T00:00:01.000000015Z, _field: "bar", _value: 6.0,_measurement: "mm"}, 
+        {_time: 2021-01-01T00:00:01.000000022Z, _field: "bar", _value: 1.2,_measurement: "mm"}, 
+        {_time: 2021-01-01T00:00:01.000000031Z, _field: "bar", _value: 0.28,_measurement: "mm"},
+    ],
+ ) |> group(columns: ["_field", "_measurement"])
```

Comment:
We probably need to add a `|> testing.load()` here. For flux, that's the equivalent of not having it, but in order for this to be an acceptance test, we need to make sure that these tests are operating against the storage parts of influxdb and cloud. Otherwise, it won't tell us if there's an issue in one of those environments.